datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -687,4 +687,7 @@ CREATE TABLE dcsa_im_v3_0.ebl_solution_provider_type (
 --- DDT-948
 ALTER TABLE dcsa_im_v3_0.equipment_event ADD utilized_transport_equipment_id uuid NULL REFERENCES dcsa_im_v3_0.utilized_transport_equipment(id);
 
+-- DDT-1058
+ALTER TABLE dcsa_im_v3_0.shipment_event ADD document_reference varchar(100);

```

Comment:
Move up before the UNION commented above...

Reply:
Also done